src/index.js

```diff
@@ -2,13 +2,25 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
+
+import About from './components/About';
+import TextForm from './components/TextForm';
 import reportWebVitals from './reportWebVitals';
+import { Router , Route , Switch } from 'react-router-dom';
+import { createBrowserHistory } from "history";
 
+const history = createBrowserHistory();
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+    <Router history={history}>
+      <Switch>
+        <Route exact path="/"  component={App} />
```

Comment:
ye ku set kiya h ? 
component = {App}

Reply:
kya component render Karna hain for that specific path woo component main pass Karna Hoga